ldms/src/sampler/Makefile.am

```diff
@@ -302,6 +302,24 @@ if ENABLE_BLOB_STREAM
 SUBDIRS += blob_stream
 endif
 
+if ENABLE_LIBZFS
+if HAVE_LIBZFS
```

Comment:
If you follow my suggestion in configure.ac, we can have just one "ENABLE_LIBZFS" (or "ENABLE_ZFS" after the name change) here. You can put all three "SUBDIRS +=" under one "if ENABLE_ZFS" block.